guides/2022-04-05-build-a-conduit-connector.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: 'How to build a Conduit Connector'
+slug: build-a-conduit-connector
+author: '@anaptfox'
+author_url: https://twitter.com/anaptfox
+author_image_url: https://pbs.twimg.com/profile_images/1004707739405205504/BpvzOr9X_400x400.jpg
+tag: ['Algolia Connector']
+description: Learn how to build a Conduit Connector by implmenting an Algolia Destination Connector.
+keywords: ['data', 'connector', 'algolia', 'conduit']
+---
+
+# How to build a Conduit Algolia Connector
+
+In this article we are going to walkthrough step by step how to build a Conduit connector. 
```

Comment:
I'm definitely not the grammar authority around here so feel free to pick and choose my suggestions 😄 
```suggestion
In this article we are going to walk through how to build a Conduit connector step by step.
```